### PR TITLE
[Beta] Upgrade Sandpack

### PR DESCRIPTION
**Pull request by gaearon**
Resubmit of https://github.com/reactjs/reactjs.org/pull/4843 with a few fixes on top.

### CODE REVIEW

**Comment by harish-sethuraman** [comment]
Did this just introduce a horizontal scroll? https://beta-reactjs-6gh2p4qfo-fbopensource.vercel.app/learn/managing-state

**Comment by gaearon** [comment]
yeah i think something here is doing this...

**Comment by gaearon** [comment]
something's wrong with the layout of long lines in sandboxes. they don't fit anymore.

File: beta/src/styles/sandpack.css [hunk 1]
```diff
@@ -2,289 +2,267 @@
...
+  overflow: visible;
+  -webkit-mask-image: -webkit-radial-gradient(
+    var(--sp-colors-surface1),
+    var(--sp-colors-surface1)
+  ); /* safest way to make all corner rounded */
+
+  border-bottom-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+}
+.sandpack .sp-layout > .sp-stack {
```
Comment (harish-sethuraman) [inline beta/src/styles/sandpack.css:83]:
```suggestion
@media screen and (max-width: 768px)
.sp-layout>.sp-stack {
    height: auto;
    min-width: 100%!important;
}

.sandpack .sp-layout > .sp-stack {
```

May be this fixes the issue? i see this is missing with this branch?

Comment (gaearon) [reply beta/src/styles/sandpack.css:89]: thanks!

**Comment by gaearon** [comment]
seems fine. let's fix forward

**Pull request merged**